### PR TITLE
refactor: Move driver-specific commands to use extensions (part2)

### PR DESCRIPTION
**Pull request by mykola-mokhnach**

### CODE REVIEW

**Comment by KazuCocoa** [comment]
will take a look later

File: appium/webdriver/extensions/android/activities.py [hunk 1]
```diff
@@ -12,21 +12,26 @@
...
+    from appium.webdriver.webdriver import WebDriver
 
 
-class Activities(CanExecuteCommands):
-    def start_activity(self: T, app_package: str, app_activity: str, **opts: str) -> T:
+class Activities(CanExecuteCommands, CanExecuteScripts, CanRememberExtensionPresence):
+    def start_activity(self, app_package: str, app_activity: str, **opts: str) -> 'WebDriver':
         """Opens an arbitrary activity during a test. If the activity belongs to
         another application, that application is started and the activity is opened.
+        deprecated:: 2.0.0
```
Comment (KazuCocoa) [inline appium/webdriver/extensions/android/activities.py:34]: Does this 2.0.0 mean this client library?
Current one is 2.9.0, so 3.0.0?

Comment (mykola-mokhnach) [reply appium/webdriver/extensions/android/activities.py:34]: I was not quite sure what to put there. I assume it should be the current version, so will change it to 2.9.0

Comment (KazuCocoa) [reply appium/webdriver/extensions/android/activities.py]: Ah, gotcha

**Review by KazuCocoa — APPROVED** [review]

**Pull request merged**